tests/test_structured_state.py

```diff
@@ -279,9 +279,6 @@ def test_canonicalise_ttn(center: Union[RootPath, Qubit]) -> None:
     ],
 )
 def test_exact_circ_sim(circuit: Circuit, algorithm: SimulationAlgorithm) -> None:
-    if algorithm in [SimulationAlgorithm.MPSxGate, SimulationAlgorithm.MPSxMPO]:
-        circuit, _ = prepare_circuit_mps(circuit)
-

```

Comment:
Some of the circuits that we're testing on had gates acting on non-adjacent qubits, so we applied `prepare_circuit_mps` to "route" these circuits, adding SWAPs. This is no longer needed and, by removing this preprocessing step, we are testing the non-adjacent 2q gate method implemented in this PR.